prow/github/types.go

```diff
@@ -46,8 +46,17 @@ const (
 	// DefaultAPIEndpoint is the default GitHub API endpoint.
 	DefaultAPIEndpoint = "https://api.github.com"
 
+	// DefaultBase is the default GitHub base endpoint.
+	DefaultBase = "https://github.com"
+
 	// DefaultGraphQLEndpoint is the default GitHub GraphQL API endpoint.
 	DefaultGraphQLEndpoint = "https://api.github.com/graphql"
+
+	// DefaultAuthEndpoint is the default Github oauth auth endpoint
+	DefaultAuthEndpoint = "https://github.com/login/oauth/authorize"
```

Comment:
When is this something aside from `DefaultBase + "/login/oauth/authorize"`?

Reply:
Oh, yeah, I don't have to add a new const here.